lib/modules/csv/model/pom.xml

```diff
@@ -56,4 +56,47 @@
     </dependency>
   </dependencies>
 
+  <build>
+    <plugins>
+      <!-- override plugin to include the jandex resource -->
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <version>${maven-bundle-plugin.version}</version>
+        <inherited>true</inherited>
+        <extensions>true</extensions>
+        <configuration>
+          <excludeDependencies>${osgi.exclude.dependencies}</excludeDependencies>
+          <instructions>
+            <Bundle-Name>${project.name}</Bundle-Name>
+            <Bundle-SymbolicName>${osgi.symbolic.name}</Bundle-SymbolicName>
+            <Bundle-DocURL>${project.url}</Bundle-DocURL>
+            <Export-Package>${osgi.export.pkg}</Export-Package>
+            <Import-Package>${osgi.import.pkg}</Import-Package>
+            <Export-Service>${osgi.export.service}</Export-Service>
+            <Implementation-Title>${project.description}</Implementation-Title>
+            <Implementation-Version>${project.version}</Implementation-Version>
+            <Embed-Dependency>${osgi.embed.dependency}</Embed-Dependency>
+            <Embed-Transitive>${osgi.embed.transitive}</Embed-Transitive>
+            <Include-Resource>${osgi.include.resource},META-INF/jandex.idx=target/classes/META-INF/jandex.idx
+            </Include-Resource>
+          </instructions>
+        </configuration>
+      </plugin>
+      <plugin>
```

Comment:
This could be moved to `PluginManagement` in parent/pom.xml with having `skip` to be true by default, maybe with a property like `jandex.skip`. Then only in children that needs jandex, set the `jandex.skip` property to true.